test/functional/tests/conftest.py

```diff
@@ -57,61 +51,53 @@ def pytest_runtest_setup(item):
     #
     # User can also have own test wrapper, which runs test prepare, cleanup, etc.
     # Then it should be placed in plugins package
+    TestRun.start(item)
+    for dut in TestRun.use_all_duts():
+        if not installer.check_if_installed():
+            continue
+        TestRun.LOGGER.info(f"CAS cleanup on {dut.ip}")
+        remove(opencas_drop_in_directory, recursive=True, ignore_errors=True)
 
-    test_name = item.name.split('[')[0]
-    TestRun.LOGGER = create_log(item.config.getoption('--log-path'), test_name)
+        try:
+            InitConfig.create_default_init_config()
+            unmount_cas_devices()
+            try:
+                casadm.stop_all_caches()
```

Comment:
isn't DRBD missing from here?

Reply:
it is - I moved it together with all other prepare stuff to testframework https://github.com/intel-innersource/frameworks.validation.opencas.test-framework/pull/52